R/helpers_knitr_Stopping.R

```diff
@@ -105,6 +113,7 @@ knit_print.StoppingAny <- function(
     x,
     ...,
     preamble = "If any of the following rules are `TRUE`:\n",
+    indent = 0L,
```

Comment:
I don't think we need to add `indent` explicitly here.  It could be handled by `...`, which is already passed to `knit_print.StoppingList`.  @danielinteractive what are your thoughts?

Reply:
yeah if we can pass through with `...` that will be better